### PR TITLE
Use manual CSF mask and improve CSF/SC figure

### PR DESCRIPTION
**Pull request by jcohenadad**
- Fixes https://github.com/shimming-toolbox/rf-shimming-7t/issues/56
- Fixes https://github.com/shimming-toolbox/rf-shimming-7t/issues/67
- Fixes https://github.com/shimming-toolbox/rf-shimming-7t/issues/66

### CODE REVIEW

**Comment by jcohenadad** [comment]
Before/After changing the CSF masks:

anim

**Review by dpapp86 — COMMENTED** [review]
I found some issues. Mainly, while I can't get the figure to show (it hangs), we are still re-sampling to C1:T2, while the extracted value is only between C3:T2

**Comment by dpapp86** [comment]
@jcohenadad added review. Nothing really major

File: data_processing.ipynb [hunk 1]
```diff
@@ -254,7 +286,6 @@
    "source": [
     "# Register *_T2starw to CoV_T2starw\n",
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply data_processing.ipynb:287]: Good point, issue opened https://github.com/shimming-toolbox/rf-shimming-7t/issues/73

File: data_processing.ipynb [hunk 2]
```diff
@@ -322,12 +336,12 @@
...
    "outputs": [],
    "source": [
-    "# Make figure of SC/CSF signal ratio from T2starw scan\n",
+    "# Make figure of CSF/SC signal ratio from T2starw scan\n",
     "\n",
     "# Go back to root data folder\n",
     "os.chdir(path_data)\n",
     "\n",
-    "def smooth_data(data, window_size=20):\n",
+    "def smooth_data(data, window_size=40):\n",
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply data_processing.ipynb:344]: yeah, I played around with this a little. The problem with increasing that too much, is that the values around C1-C2 are quite different from the rest of the cord, therefore with too much smoothing, the C1-C2 values 'leak' too much to the C3-C4 levels. And what we want to show with this figure is how 'good' the C3-T2 region is, hence we want to minimize the contamination of C1-C2. 

Alternatively, we could crop out C1-C2 before doing the smoothing... hum. Discussion opened: https://github.com/shimming-toolbox/rf-shimming-7t/issues/74

File: data_processing.ipynb [hunk 3]
```diff
@@ -312,7 +326,7 @@
     "        fname_result_sc = os.path.join(path_results, f\"{subject}_acq-{shim_method}_T2starw_label-SC.csv\")\n",
     "        !sct_extract_metric -i {subject}_acq-{shim_method}_T2starw_{file_suffix}.nii.gz -f {subject}_acq-CoV_T2starw_crop_seg.nii.gz -method wa -vert 3:9 -vertfile {subject}_acq-CoV_T2starw_crop_seg_labeled.nii.gz -perslice 1 -o {fname_result_sc}\n",
     "        fname_result_csf = os.path.join(path_results, f\"{subject}_acq-{shim_method}_T2starw_label-CSF.csv\")\n",
-    "        !sct_extract_metric -i {subject}_acq-{shim_method}_T2starw_{file_suffix}.nii.gz -f {subject}_acq-CoV_T2starw_crop_CSFseg.nii.gz -method wa -vert 3:9 -vertfile {subject}_acq-CoV_T2starw_crop_seg_labeled.nii.gz -perslice 1 -o {fname_result_csf}"
+    "        !sct_extract_metric -i {subject}_acq-{shim_method}_T2starw_{file_suffix}.nii.gz -f {subject}_acq-CoV_T2starw_crop_label-CSF_seg.nii.gz -method wa -vert 3:9 -vertfile {subject}_acq-CoV_T2starw_crop_seg_labeled.nii.gz -perslice 1 -o {fname_result_csf}"
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply data_processing.ipynb:329]: oh gosh! good catch 🙏 . Issue: https://github.com/shimming-toolbox/rf-shimming-7t/issues/75

File: data_processing.ipynb [hunk 4]
```diff
@@ -375,7 +389,7 @@
     "        data_csf = df['WA()']\n",
     "        \n",
     "        # Compute ratio\n",
-    "        data_sc_csf_ratio = data_sc / data_csf\n",
+    "        data_sc_csf_ratio = data_csf / data_sc\n",
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply data_processing.ipynb:392]: The reason for this change is commented in this commit message: https://github.com/shimming-toolbox/rf-shimming-7t/pull/69/commits/099728c56087bea912e4fdb64f6adc5f8746b0f6

**Pull request merged**